weepickle/test/src/com/rallyhealth/weepickle/v1/AdvancedTests.scala

```diff
@@ -1,7 +1,9 @@
 package com.rallyhealth.weepickle.v1
 import utest._
 import acyclic.file
+import com.rallyhealth.weejson.v1.jackson.{FromJson, ToJson}
 import com.rallyhealth.weepickle.v1.TestUtil.rw
+import com.rallyhealth.weepickle.v1.WeePickle.{FromScala, ToScala}
```

Comment:
is this extraneous? I don't see any changes to this file besides removing a blank line...